module/applications/actor/character-sheet.mjs

```diff
@@ -24,11 +24,10 @@ export default class ActorSheet5eCharacter extends ActorSheet5e {
 
     // Resources
     context.resources = ["primary", "secondary", "tertiary"].reduce((arr, r) => {
-      const res = context.actor.system.resources[r] || {};
-      res.name = r;
-      res.placeholder = game.i18n.localize(`DND5E.Resource${r.titleCase()}`);
-      if (res && res.value === 0) delete res.value;
-      if (res && res.max === 0) delete res.max;
```

Comment:
Now that we've broken the reference and are (presumably) no longer mutating real actor data, it should be fine to still have these, no? Is it no longer relevant to the sheet logic?

Reply:
From what I gather either seems fine. Either the value is 0 and gets deleted and a placeholder (which is '0') gets shown, or we show the value (which may we '0').